src/ydata_profiling/report/presentation/flavours/html/templates/table.html

```diff
@@ -18,11 +18,11 @@
                 <tr{% if 'alert' in row and row['alert'] %} class="alert"{% endif %}>
                     <th>{{ row['name'] }}</th>
                     {% if row['value'].__class__.__name__ == 'list' %}
-                        {% for value in row['value'] %}
-                            <td>{{ value }}{% if loop.last and 'hint' in row %} {{ row['hint'] }}{% endif %}</td>
+                        {% for value in row['value'] +%}
```

Comment:
I think the '+' sign should not be there? I'm recieving the error 'TemplateSyntaxError: unexpected 'end of statement block'' when running this code.

Reply:
This breaking the render to html profile...

Reply:
It's true, removing it fixed the HTML render

Reply:
This error was fixed in https://github.com/ydataai/ydata-profiling/pull/1491 and will be added to the next release